helm/ph-ee-engine/values.yaml

```diff
@@ -442,8 +442,8 @@ mpesa:
       client_key: "0pLxbN83FrOl5Nd0Fh9Zi5BQlMxSL2n5"
       client_secret: "YzuGNoJxeub8ZC6d"
       pass_key: "bfb279f9aa9bdbcf158e97dd71a467cd2e0c893059b10f78e6b72ada1ed2c919"
+  paybill_accountHoldingInstitutionId: "default"
```

Comment:
keep this as . format and make sure that this variable is added to the corresponding service